.github/workflows/wfc_backport.yml

```diff
@@ -0,0 +1,117 @@
+on:
+  workflow_call:
+    inputs:
+      label:
+        description: "The label to use to indicate backport"
+        type: string
+        default: "backport"
+        required: false
+      branchFilePath:
+        description: "A file that contains the list of active branches"
+        type: string
+        default: "active-branches.json"
```

Comment:
So we always backport to all branches in this file? Maybe in the future, we could allow specifying branches to which we want to backport. I can imagine that some changes shouldn't be backported to older versions because those features weren't available.